### PR TITLE
Add Interlogix Ultrasync Alarm System Integration

### PR DESCRIPTION
**Pull request by caronc**
## Proposed change
Added documentation for new integration: Interlogix UltraSync Hub

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  [core/42549](https://github.com/home-assistant/core/pull/42549)
- Link to parent pull request in the Brands repository: [brands/1994](https://github.com/home-assistant/brands/pull/1994)
- This PR fixes or closes issue: n/a

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please rebase this pull request to the `next` branch

**Comment by caronc** [comment]
@klaasnicolaas Thanks for your feedback; done! :+1:

File: source/_integrations/ultrasync.markdown [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Interlogix UltraSync Hub
+description: Instructions on how to integrate your Interlogix UltraSync Hub within Home Assistant.
+ha_category:
+  - Alarm
+logo: informix.png
```
Comment (klaasnicolaas) [inline source/_integrations/ultrasync.markdown]: Please add a logo to our [brands repository](https://github.com/home-assistant/brands), you can remove the line here

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/ultrasync.markdown (hunk 1 above)
Comment (caronc) [reply source/_integrations/ultrasync.markdown]: Done; updated comments. Here is a [quick link to brands/1994](https://github.com/home-assistant/brands/pull/1994)
You also brought forth another bug, it shouldn't have read `informix.png`, but `interlogix.png` instead.  Sorry about that; this is also fixed in latest push to this branch.

Comment (frenck) [reply source/_integrations/ultrasync.markdown]: You'll need to remove the `logo` manifest from this PR.

File: source/_integrations/ultrasync.markdown [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ha_domain: ultrasync
+---
+
+The `ultrasync` platform will allow you to monitor and control your Interlogix UltraSync Hub from within Home Assistant and setup automation.
+
+## Configuration
+
+Go to the integrations page in your configuration and click on new integration -> UltraSync.
+
+**Note**: You can only be logged into the ZeroWire/UltraSync hub with the same user once; a subsequent login with the same user logs out the other. Since Home Assistant (HA) actively polls and maintains a login session to this Hub, it can prevent you from being able to log into at the same time elsewhere (via it's website). It is strongly recommended that you create a second user account on your Hub dedicated for just HA.
```
Comment (frenck) [inline source/_integrations/ultrasync.markdown]: Don't abbreviate HA.

File: source/_integrations/ultrasync.markdown [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+logo: ultrasync.png
+ha_iot_class: Local Polling
+ha_release: 0.119
+ha_config_flow: true
+ha_codeowners:
+  - '@caronc'
+ha_domain: ultrasync
+---
+
+The `ultrasync` platform will allow you to monitor and control your Interlogix UltraSync Hub from within Home Assistant and setup automation.
```
Comment (frenck) [inline source/_integrations/ultrasync.markdown]:
```suggestion
The Ultrasync integration will allow you to monitor and control your Interlogix UltraSync Hub from within Home Assistant and setup automation.
```

File: source/_integrations/ultrasync.markdown [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+## Configuration
+
+Go to the integrations page in your configuration and click on new integration -> UltraSync.
+
+**Note**: You can only be logged into the ZeroWire/UltraSync hub with the same user once; a subsequent login with the same user logs out the other. Since Home Assistant (HA) actively polls and maintains a login session to this Hub, it can prevent you from being able to log into at the same time elsewhere (via it's website). It is strongly recommended that you create a second user account on your Hub dedicated for just HA.
+
+## Sensor
+
+This component will create these sensors in the format of `{ultrasync_hubname}_{sensor}`;  The below example assumes you accept the default name of `UltraSync` (which is still represented in lowercase):
```
Comment (frenck) [inline source/_integrations/ultrasync.markdown]: The names entity names can be changed by the user, right? They are also visible in the device overview, right?

So the device entity name hint should not be needed.

Comment (frenck) [inline source/_integrations/ultrasync.markdown]: Additionally, we don't use double spacing.

File: source/_integrations/ultrasync.markdown [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+- `ultrasync_area1state`: The Area 1 State
+- `ultrasync_area2state`: The Area 2 State
+- `ultrasync_area3state`: The Area 3 State
+- `ultrasync_area4state`: The Area 4 State
+
+There are several states each sensor can be at, but usually they will be one of the following: `Unknown`, `Ready`, `Not Ready`, `Armed Stay`, and `Armed Away`.  The `Unknown` state is assigned to sensors that are not reporting; they usually sit in the spots of the Area's you're not monitoring.
```
Comment (frenck) [inline source/_integrations/ultrasync.markdown]:
```suggestion
There are several states each sensor can be at, but usually, they will be one of the following: `Unknown`, `Ready`, `Not Ready`, `Armed Stay`, and `Armed Away`. The `Unknown` state is assigned to sensors that are not reporting; they usually sit in the spots of the areas you're not monitoring.
```

File: source/_integrations/ultrasync.markdown [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+## Event Automation
+
+When an Zone or Sensor changes it's state an event usable for automation is triggered on the Home Assistant Bus.
```
Comment (frenck) [inline source/_integrations/ultrasync.markdown]: Suggestion: add support for device automations to the integration?

```suggestion
When a zone or sensor changes its state an event is triggered. Those events are usable for automations.
```

File: source/_integrations/ultrasync.markdown [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+Possible events are:
+
+- `ultrasync_sensor_update`: The event includes the sensor no, name, and new status it changed to.
+- `ultrasync_area_update`: The event includes the area no, name, and new status it changed to (if it did). **Note**: Area's are sent periodic heartbeats in which case the state will not change at all.
+
+Example automation to send a message via [Apprise](https://www.home-assistant.io/integrations/apprise/) on a sensor change in your home:
```
Comment (frenck) [inline source/_integrations/ultrasync.markdown]:
```suggestion
Example automation to send a message via [Apprise](/integrations/apprise/) on a sensor change in your home:
```

File: source/_integrations/ultrasync.markdown [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+{% endraw %}
+
+## Services
+
+Available services:
+
+- `stay`: Set alarm scene to Stay Mode
+- `away`: Set alarm scene to Away Mode (fully activate Alarm)
+- `disarm`: Disarm the alarm
```
Comment (frenck) [inline source/_integrations/ultrasync.markdown]: Aren't these services dictated by the alarm control panel entity component?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by caronc** [comment]
I pushed a new change and that some how triggered a close on this branch :man_shrugging:

**Pull request closed without merging**